### DISCUSSION TITLE
OAUTH2 SMTP Authentication

### DISCUSSION THREAD

**Discussion by alexspanos-wide** [globaleaks/globaleaks-whistleblowing-software#4090] [Ideas]
Hello,

We see that right now, gloabaleaks supports only basic SMTP Auth. Are there any plans to also implement Oauth2 authentication?

thank you.

**Comment by evilaliv3** [comment]
Thank you for the question @alexspanos-wide 

I dont see particular advantages on a machine to machine communication but yes, many ask for this so yes i think sooner or later we could implement it; Do you have possibility to provide a patch yourself?
